frame/evm/src/lib.rs

```diff
@@ -563,6 +563,11 @@ impl<T: Config> Module<T> {
 		account.nonce == U256::zero() && account.balance == U256::zero() && code_len == 0
 	}
 
+	pub fn is_contract_code_empty(address: &H160) -> bool {
```

Comment:
is_contract_code_empty(address: &H160) -> 

empty(address: &H160) or empty_address(address: &H160)

It seems this function is validating address not the contract

Reply:
In the Ethereum system, an address can represent both a user account and a contract account. The difference between them is whether they have contract code or not. Here is represent contract account.